bin/cron.js

```diff
@@ -158,7 +408,8 @@ async function updateModuleProgress(
   try {
     let score = 0;
     const moduleProgress = {};
-    submissions.map((submission) => {
+    for (const submission of submissions) {
```

Comment:
Do you know why there it's showing a diff of changes that were already merged? I made this change a while ago. Did you pull overhaul after you committed or something?

Reply:
Yea I'm not sure either, I think I may have made this branch off an old commit a bit ago